Classes/Private/Manager/Database/OCTDBMessageCall.h

```diff
@@ -0,0 +1,18 @@
+//
+//  OCTDBMessageCall.h
+//  objcTox
+//
+//  Created by Chuong Vu on 5/14/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "RLMObject.h"
+#import "OCTMessageCall.h"
+
+@interface OCTDBMessageCall : RLMObject
+
+@property NSDate *endTime;
```

Comment:
http://realm.io/docs/objc/latest/#current-limitations
> NSDate is truncated to the second
> 
> Persisting an NSDate with a fractional number of seconds will truncate the date to the second. A fix for this is in progress. See GitHub issue #875 for more details. In the mean time, you can store NSTimeInterval properties with no loss in precision.

I'm not sure if it is so important to have precise `endTime`, but in all other cases I'm using NSTimeInterval instead of NSDate for Realm objects.